service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -944,5 +955,599 @@
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET] has more baggage than an airport carousel.",
+    "targetableSsml":"[TARGET] has more baggage than an airport carousel."
+  },
+  {
+    "quipGroup":"ILLITERATE",
+    "text":"Most of our conversations go over your head because you have a hard time reading between the lines. I suppose that's to be expected from someone who's illiterate.",
+    "ssml":"Most of our conversations go over your head because you have a hard time reading between the lines. I suppose that's to be expected from someone who's illiterate.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"RUNS_IN_FAMILY",
+    "text":"I want to say that obesity runs in your family. But it's pretty clear, no one runs in your family.",
+    "ssml":"I want to say that obesity runs in your family. But it's pretty clear, no one runs in your family.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"COLD_BLOODED",
+    "text":"You're so cold blooded you could be a lizard. It'd probably explain your scaly face and languid lifestyle.",
+    "ssml":"You're so cold blooded you could be a lizard. It'd probably explain your scaly face and languid lifestyle.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"SUPER_GLUE_RELATIONSHIP",
+    "text":"Using super glue is the only way you'll ever have a relationship where they stick around.",
+    "ssml":"Using super glue is the only way you'll ever have a relationship where they stick around.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"Using super glue is the only way [TARGET] will ever have a relationship where they stick around.",
+    "targetableSsml":"Using super glue is the only way [TARGET] will ever have a relationship where they stick around."
+  },
+  {
+    "quipGroup":"DOWNHILL_LIFE",
+    "text":"Your life has picked up speed lately. I guess that's what happens when everything is going downhill.",
+    "ssml":"Your life has picked up speed lately. I guess that's what happens when everything is going downhill.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET]'s life has picked up speed lately. I guess that's what happens when everything is going downhill.",
+    "targetableSsml":"[TARGET]'s life has picked up speed lately. I guess that's what happens when everything is going downhill."
+  },
+  {
+    "quipGroup":"SELF_ESTEEM_DECEPTION",
+    "text":"Your self-esteem is only possible due to your talent of self-deception.",
+    "ssml":"Your self-esteem is only possible due to your talent of self-deception.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"EAT_GARBAGE",
```

Comment:
Check if we already have a "you are what you eat" reference, and make sure the quipGroups match.